### PR TITLE
EUI-2140-Duplicate Main Landmarks

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2140

### Change description ###
Only one main landmark should be available

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/register/components/notification-banner/notification-banner.component.html [hunk 1]
```diff
@@ -1,8 +1,6 @@
-<main id="content" role="main">
-  <div aria-labelledby="error-summary-title" class="govuk-error-summary">
-    <h2 class="govuk-error-summary__title">There is a problem</h2>
-    <div class="govuk-list govuk-error-summary__list">
-    <a href="/register-org/register/check#h1"> <ng-content></ng-content></a>
-    </div>
+<div aria-labelledby="error-summary-title" class="govuk-error-summary">
+  <h2 class="govuk-error-summary__title">There is a problem</h2>
```
Comment (paul-graham) [inline src/register/components/notification-banner/notification-banner.component.html:2]: I think this needs to have `id="error-summary-title"` or the `aria-labelledby` property on the surrounding `<div>` will have an accessibility issue.

**Review by paul-graham — CHANGES_REQUESTED** [review]
`aria-labelledby` concern.

File: src/register/components/notification-banner/notification-banner.component.html (hunk 1 above)
Comment (RiteshHMCTS) [reply src/register/components/notification-banner/notification-banner.component.html]: id="error-summary-title" has been added

**Review by paul-graham — APPROVED** [review]
Thanks for sorting out that `aria-labelledby` concern, @RiteshHMCTS. 👍

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "ng": "ng",
     "start:ng": "ng serve --port 3000 --host 0.0.0.0 --proxy-config proxy.config.json",
     "build:ng": "ng build",
-    "test:a11y": "webdriver-manager update --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
+    "test:a11y": "webdriver-manager update --gecko=false --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
```
Comment (adnanakgun) [inline package.json:36]: Why are we making tis change?

Comment (RiteshHMCTS) [reply package.json:36]: build was failing; I got this solution from Vamshi

**Review by adnanakgun — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]
Given the explanation about `--gecko=false` and that nothing else has changed, re-approving. 👍

**Pull request merged**